modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,538 @@
+  --member="<redpanda-console-service-account-email>" \
+  --role='projects/<project-id>/roles/redpanda_console_secret_manager_role'
+```
+
+.. Redpanda Connectors SA:
++
+```bash
+gcloud iam service-accounts create redpanda-connectors \
+  --display-name="Redpanda Connectors Service Account"
+```
+
+* Connectors custom role permissions:
++
+```bash
+gcloud iam roles create redpanda_connectors_role \
+  --project="<project-id>" \
+  --title="Redpanda Connectors Custom Role" \
+  --permissions=resourcemanager.projects.get,secretmanager.versions.access
+```
+
+* Project bindings:
++
+```bash
+gcloud iam service-accounts add-iam-policy-binding <redpanda-connectors-service-account-email> \
+  --member="<redpanda-connectors-service-account-email>" \
+  --role='projects/<project-id>/roles/redpanda_connectors_role'
+```
+
+* GKE SA:
++
+```bash
+gcloud iam service-accounts add-iam-policy-binding <gke-service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[cert-manager/cert-manager]"
+gcloud iam service-accounts add-iam-policy-binding <gke-service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[external-dns/external-dns]"
+```
+
+. Because some resources can only be created after the Redpanda ID is known, xref:./create-byoc-cluster-gcp.adoc[create a BYOC cluster] in the https://cloudv2.redpanda.com[Redpanda Cloud UI^] to get the Redpanda ID.
++
+IMPORTANT: On the Network page, select the *Customer-managed* network connection type, and enter the network, service account, and storage bucket information you created. Click *Next*, but before running the `rpk` command on the Deploy page, note the redpanda-id (for example, `cisld88gfi809ee1qjcg`). You finish the cluster deployment in a later step.  
++
+. Use the `redpanda-id` to bind the service accounts with the following roles:
+
+* Redpanda cluster SA:
++
+```bash
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/rp-<redpanda-id>]"
+```
+
+* Redpanda Console SA:
++
+```bash
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/console-<redpanda-id>]"
+```
+
+* Connectors SA:
++
+```bash
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda-connectors/connectors-<redpanda-id>]"
+```
+
+== Deploy the BYOC cluster
```

Comment:
suggestion: add section titled "Validate Configurations with rpk", and ask user to run `rpk byoc gcp validate` to validate the configurations before deploy cluster.

Reply:
Thanks @paulzhang97! Please confirm if this edit is correct: https://deploy-preview-23--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/vpc-byo-gcp/#validate-configuration

Reply:
The command should be
```
rpk byoc gcp apply --validate-only --redpanda-id=<redpanda-id> --project_id=<service-project-id>
```

Reply:
@paulzhang97 There is another command that can be used before the redpanda id is created. With this one if there are any issues, it's too late to correct them.

Reply:
@frenchfrywpepper Good point. Let's use that command. I don't know the exact usage. Could you share?

Reply:
hmm, I just tried it out and I think there is an issue. I have re-opened https://github.com/redpanda-data/cloudv2/issues/8980 and I will investigate.